at_secondary/at_secondary_server/config/generate_config.py

```diff
@@ -0,0 +1,49 @@
+"""
+Generates the config.yaml from the config-base.yaml
+The default values in the configurations can be overridden by creating an
+config-<environment>.properties file
+The file accepts an optional parameter <environment>; defaults to production.
+Example:
+    Generating config.yaml for development.
+        create an application properties file as 'config-development.properties'
+        Run the generate_config.py passing 'development' as command line argument.
+            python3 generate_config.py -e development
+"""
+import argparse
+import yaml
+from jproperties import Properties
+
+# Initialize parser
+parser = argparse.ArgumentParser()
+# Adding optional argument
+parser.add_argument("-e", "--env", default='production')
+
+# Read arguments from command line
+environment = parser.parse_args().env
```

Comment:
It would help to have some console.writes on for what environment we are building the config

Reply:
Sure Jagan, Will add a print statement.